common-theme/layouts/partials/map-common.html

```diff
@@ -1,34 +1,52 @@
-{{/*
-Expects a slice[dict] param containing an ordered list of dicts with the following keys, one per section in the map to render:
-* title (string) - The heading to display for the section.
-* values (slice[Page]) - A slice of Pages, one per page in the section which should be listed.
+{{/* Expects a slice[dict] param containing an ordered list of dicts with the following keys, one per section in the map to render:
+  * title (string) - The heading to display for the section.
+  * values (slice[Page]) - A slice of Pages, one per page in the section which should be listed.
+  * metadata (slice[dict[string, string]]) - Ordered key-value pairs of extra metadata to display about the section. Each dict should contain a key named "name" and a key named "value".
 */}}
 
+
 <div class="c-map">
-<h1 class="e-heading c-map__start is-none--lt-container">👉🏾</h1>
-{{ range $i, $section := index . "sections" }}
-  {{ $sectionTitle := index $section "title" }}
-  {{ $sectionValues := index $section "values" }}
-  <section class="c-map__block">
-    <h2 class="e-heading c-map__title">
-      📍{{ $i }}:
-      {{ $sectionTitle }}
-    </h2>
-    <ol class="c-map__timeline">
-      {{/* Check if a menu for the current map section exists */}}
-      {{ if $sectionValues }}
-        {{/* Range over the items in the menu */}}
-        {{ range $iterator, $module := $sectionValues }}
-          <li
-            class="c-map__stop"
-            style="--layer:{{ sub (len $sectionValues) (add $iterator 1) }}">
-              {{ partial "card.html" $module }}
-          </li>
-        {{ end }}
+  <h1 class="e-heading c-map__start is-none--lt-container">👉🏾</h1>
+  {{ range $i, $section := index . "sections" }}
+    {{ $sectionTitle := index $section "title" }}
+    {{ $sectionTermPageLink := index $section "termPageLink" }}
+    {{ $sectionValues := index $section "values" }}
+    <section class="c-map__block">
+      {{ if $sectionTermPageLink }}
```

Comment:
I think if it's a term we flip to this disclosure view and also link to the term

Reply:
But as per comment I'm not wedded to this!